src/components/Claims/README.md

```diff
@@ -0,0 +1,38 @@
+## Overview
+
+[#TODO]
+
+This folder contains the code for User Claims - a demo leveraging Ably's [user claims](https://ably.com/docs/api/realtime-sdk/authentication#ably-jwt) feature to allow users to delete their own or other's messages depending on privilege level.
+
+Try out the [live demo](https://examples.ably.dev/claims) of the User Claims.
+
+## Running locally
+
+- Fork or clone the [Examples Repo](https://github.com/ably-labs/realtime-examples).
+- Sign up for an account on [Ably](https://ably.com/sign-up?utm_source=ably-labs&utm_medium=github&utm_campaign=user-claims) and get an API KEY.
+- Rename `.env.example` to `.env` and fill in your API KEY in the `VITE_ABLY_KEY` environment variable.
+- Run `yarn` or `npm install` to install dependencies.
+- Run `yarn dev` or `npm run dev` and go to http://localhost:5173/emoji-reactions
+
+## Runtime Requirements
+
+- [Node.js](https://nodejs.org/en/)
+- Ably API Key
+- [Yarn](https://yarnpkg.com/) (optional)
+
+## Resources
+
+- Learn more about how to build your own [emoji reactions](https://ably.com/examples/emoji-reactions?utm_source=ably-labs&utm_medium=github&utm_campaign=user-claims).
```

Comment:
This should be claims specific, right? (It has emoji reactions instead.)

Reply:
Fixed https://github.com/ably-labs/realtime-examples/pull/37/commits/5b7f7d40c9d685832593110bcc1e0f5456448090